cmd/protoc-gen-connect-go/main.go

```diff
@@ -96,6 +96,17 @@ func main() {
 	)
 }
 
+func needsWithIdempotency(file *protogen.File) bool {
+	for _, service := range file.Services {
+		for _, method := range service.Methods {
+			if methodIdempotency(method) == connect.IdempotencyNoSideEffects {
```

Comment:
Since we're also going to plumb through IdempotencyIdempotent, I think we want `!= connect.IdempotencyUnknown` here.

Reply:
Fixed in 77192c355c41cb3d5b12f7c0929edb6378500f2c.